packages/split-view/package.json

```diff
@@ -18,7 +18,7 @@
         "lit-element",
         "lit-html"
     ],
-    "version": "0.1.0",
```

Comment:
This will change on its own later based on your usage of the `fix` syntax in your commit. Please revert.

Reply:
done, reverted.